concordium-node/src/bin/cli.rs

```diff
@@ -62,30 +62,9 @@ async fn main() -> anyhow::Result<()> {
         instantiate_node(&conf, &mut app_prefs, stats_export_service, regenesis_arc.clone())
             .context("Failed to create the node.")?;
 
-    #[cfg(feature = "instrumentation")]
```

Comment:
I moved this until after starting the thread in line 129 in the PR. We should perhaps discuss whether this is problematic.